src/qibolab/instruments/zhinst.py

```diff
@@ -337,6 +323,12 @@ def __init__(
         self.results = None
         "Zurich experiment definitions"
 
+        self.bounds = Bounds(
+            waveforms=int(4e4),
+            readout=250,
+            instructions=int(1e6),
```

Comment:
Where do these numbers come from. I believe ZI limitations are much larger than these numbers. 

Additionally, different ZI instruments have different limitations. Our `Zurich` class does not represent a single instrument, but a collection of ZI instruments. So the concept of bounds is not straightforwardly applicable to this class. The best we can do for now is for each property take the minimum across all possible ZI instruments.

Reply:
In principle, we should be able to serialize the configuration for every instrument in the platform. They should just live in nested sections.

Reply:
https://github.com/qiboteam/qibolab/blob/adfc27d021bd3b8e32aec8ce6ebbc149fe7fe489/src/qibolab/serialize.py#L174